public/index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <link rel="stylesheet" href="style.css" />
+    <title>Signalk charts-addin</title>
+</head>
+
+<body>
+
+    <fieldset>
+        <legend>Seed charts for offline usage</legend>
```

Comment:
or _Seed charts to prefetch tiles for offline usage_. I think the term _seeding_ is widely used for this, but for a layperson _prefetching tiles_ may give the right idea more quickly?